docs(icon): add link to theming icons section (#1571)
nuxt/ui: docs/content/2.components/icon.md

> You won't be able to use any icon in the `name` prop here as icons are bundled using [egoist/tailwindcss-icons](https://github.com/egoist/tailwindcss-icons), read more about this in [Theming](/getting-started/theming#icons).

Reviewer: ```suggestion
You won't be able to use all icons in the `name` prop here as icons are bundled using [egoist/tailwindcss-icons](https://github.com/egoist/tailwindcss-icons).
```

::

::callout{icon="i-heroicons-light-bulb"}

> Don't forget to specify the icon pack that you are using in `nuxt.config.ts`! See [here](/getting-started/theming#collections) for more information.

Reviewer: ```suggestion
Don't forget to install and specify the icon collections you need in your `nuxt.config.ts`, read more about this in [Theming](/getting-started/theming#icons).
```
